datacube_utilities/query.py

```diff
@@ -23,10 +39,10 @@ def create_base_query(aoi, res, output_projection, aoi_crs, dask_chunks):
     query = {
```

Comment:
This query dictionary could be a pyhon data class with a method to_dict for serialization, if it is a dictionary that it is used many times across the code ?